valhalla/baldr/traffictile.h

```diff
@@ -31,6 +32,9 @@ constexpr uint16_t INVALID_SPEED_AGE_BUCKET = 0;
 constexpr uint16_t MAX_SPEED_AGE_BUCKET = 15;
 constexpr uint16_t SPEED_AGE_BUCKET_SIZE = 2; // 2 minutes per bucket
 
+// Traffic speeds are encoded as 8 bits in `Speed` below
+constexpr uint32_t MAX_TRAFFIC_SPEED_KPH = 255;
```

Comment:
Yea, I was struggling with this. I wanted to avoid pulling in the graphconstants initially in order to avoid keep the C-interface a single file and instead settled with the static assert that the two were the same value.